stdlib/_collections_abc.pyi

```diff
@@ -1,5 +1,8 @@
+import sys
+from types import FunctionType as FunctionType, MappingProxyType as mappingproxy  # undocumented
```

Comment:
If they're undocumented, why are we re-exporting them here?

Reply:
I was going according to the principle of keeping the stubs as close as possible to the runtime definition; `from _collections_abc import mappingproxy` works fine at runtime, even if it's a *pretty weird* thing to be doing. But, definitely don't want to die on this hill! I'll take them out if you think it's unnecessary.

Reply:
Cool, I'll remove them

Reply:
Best to remove `generator`, `coroutine` and `async_generator` as well, do you think?

Reply:
Yes